packaging/package_all.sh

```diff
@@ -33,56 +38,82 @@ function _move_debs() {
   if ls ../*.deb 1> /dev/null 2>&1; then
     mv ../*.deb ${SCRIPT_PATH}/${DEBS_OUTPUT_DIR}/
   fi
-  if ls ../*.ddeb 1> /dev/null 2>&1; then
-    mv ../*.ddeb ${SCRIPT_PATH}/${DEBS_OUTPUT_DIR}/
-  fi
 }
 
 function _execute_build() {
+#  bloom-generate rosdebian --os-name ubuntu --os-version focal --ros-distro foxy --place-template-files
   bloom-generate rosdebian --os-name ubuntu --os-version focal --ros-distro foxy -i "${GIT_VER}"
   sed -i 's/^\tdh_shlibdeps.*/& --dpkg-shlibdeps-params=--ignore-missing-info/g' debian/rules
   fakeroot debian/rules clean
   fakeroot debian/rules "binary --parallel"
 }
 
+function _make_deb() {
+  local pkg=$1
+  cat <<- EOF
+  *********************************************************
+  Building deb package ${pkg} 
+  *********************************************************
+EOF
+  ./package.sh
+  _move_debs
+  echo "Done."
+}
+
 function _make_ros_deb() {
-  echo "Creating deb package ${1} ..."
+  local pkg=$1
+  cat <<- EOF
+  *********************************************************
+  Building ROS deb package ${pkg} 
+  *********************************************************
+EOF
   GIT_VER=0~dirty$(git log --date=format:%Y%m%d --pretty=~git%cd.%h -n 1)
-  build_dir=$(mktemp -d)
-  cp -r . ${build_dir}/package
-  pushd ${build_dir}/package > /dev/null
   _execute_build
   _move_debs
-  popd > /dev/null
-  rm -rf ${build_dir}
   echo "Done."
```

Comment:
Can we keep this? Does this work in docker? The idea is to do builds outside sources so the sources do not get polluted with build files.

Reply:
we could update gitignore files to filter build files from local workspace, so that it doesn't look so messy. IMO it would be good to keep build files in workspace, because it should make following builds faster.

Reply:
It is ok for me to add more entries to gitignore.